modules/index/src/main/java/org/apache/ignite/internal/index/IndexManagementUtils.java

```diff
@@ -263,4 +265,31 @@ static boolean isPrimaryReplica(ReplicaMeta primaryReplicaMeta, ClusterNode loca
         return localNode.id().equals(primaryReplicaMeta.getLeaseholderId())
                 && timestamp.compareTo(primaryReplicaMeta.getExpirationTime()) < 0;
     }
+
+
+    /**
+     * Returns the catalog version in which the existing index created.
+     *
+     * <p>NOTE: Can be used outside of catalog/metastore events.</p>
```

Comment:
A note to myself, because when working with a catalog I ask myself questions about the safety of working with its versions. Here I will know that I can safely call this method anywhere and not forget about it.

Reply:
I'm not sure that leaving notes for yourself in documentation is a good idea. It is also inconsistent, because all other methods don't specify that they must be called in some event threads

Reply:
Okay I deleted it.